bedrock/base/templates/includes/protocol/footer/footer.html

```diff
@@ -6,6 +6,130 @@
 
 {% set utm_params = 'utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer' %}
 
+{% if switch('brand-refresh') %}
```

Comment:
**Issue/question:** is there a plan for how this new footer is going to roll out? I understand we're going to translate it for all locales, but right now if this switch was enabled it would show for all languages even though many might still be in English.

Is the plan to try and get as many translations as we can before enabling the new footer, or should we show the new footer for [only locales that have translated it](https://bedrock.readthedocs.io/en/latest/l10n.html#the-ftl-has-messages-helper-function)?

Reply:
I am not sure about the rolling out plan. I will check with Stephanie when she is back online

Reply:
Looks like last time we did this we launched English only and then gave the languages several weeks to translate before opening it up further. So, for now, let's do that this time too.

With means we probably need a version of the current nav with the new logo in it... I will create a separate issue for that.